My-PR-Contribution-Practice.md

```diff
@@ -46,3 +46,7 @@ ________________________________________________________________________________
 
 - [Udeagbala Anthony](https://github.com/izudada)
   <p>Bio: A growing python developer, more interested in building web applications with python. </p>
+
+- [Damilare Oyedeji](https://github.com/fuglydami)
```

Comment:
Thought this is already in alphabetical order. No? if not, please how do i make the changes, i'm new to open source.

Reply:
Okay, your name starts with the word **D**amilare.
Alphabetically just means that put your name in the order of ABC.
- You are putting your name behind **U**deagbala which is not meant to be so
- Your name should be up, after any name that starts with the word **C** if there is none, then check **B** if there is none, there is **A** there.